libs/common/include/launchdarkly/context_builder.hpp

```diff
@@ -79,24 +92,27 @@ class ContextBuilder final {
                                                      std::string key);
 
     /**
-     * Build a context. This should only be called once, because
-     * the contents of the builder are moved into the created context.
+     * Start updating an existing kind.
      *
-     * After the context is build, then this builder, and any kind builders
-     * associated with it, should no longer be used.
+     * @param kind The kind to start updating.
```

Comment:
same here

I think the reason I find the current wording a little confusing is because in the UI a context kind is an entity that you can create and define (Contexts list --> Kind tab --> Create kind). It's separate from any context where it's used, and this is definitely not working with that.

Reply:
Yeah, adding a "org" to a context is different than adding an "org" to your launchdarky account. The "org" you put into a context has attributes, and you cannot even make it without any attributes.

Reply:
Both are kinds though.

Reply:
Yeah I think got hung up on the two uses of "kind":
 * the entity you add to your LD account, can (but don't have to) use when defining a new context in the SDK, and can get a list of using the REST API, etc.
 * the thing that, as you say above, happens to be a context, like the address happens to be a string, or the name is a string